components/QuickLinks.vue

```diff
@@ -1,38 +1,37 @@
 <template>
   <div class="quicklinks-cards">
-    <div v-for="quickLink in quickLinks" :key="quickLink.name">
-      <a :href="quickLink.link" target="_blank">
-        <v-card class="quicklinks-card">
-          <v-stack
-            direction="column"
-            sm-direction="column"
-            align="start"
-            gap="1.25rem"
-            sm-gap="1rem"
-          >
-            <v-text class="quicklinks-title" :weight="700" line-height="1.5">
-              {{ quickLink.title }}
-            </v-text>
-            <v-stack
-              direction="row"
-              align="center"
-              justify="center"
-              gap=".5rem"
-            >
-              <v-text
-                class="quicklinks-link-text"
-                :weight="400"
-                line-height="1.4"
-                color="secondary"
-              >
-                {{ quickLink.linkText }}
-              </v-text>
-              <v-image class="quicklinks-arrow" path="images/arrow-right.svg" />
-            </v-stack>
-          </v-stack>
-        </v-card>
-      </a>
-    </div>
+    <v-card
+      v-for="quickLink in quickLinks"
+      :key="quickLink.name"
+      class="quicklinks-card"
+    >
+      <v-stack
+        direction="column"
+        sm-direction="column"
+        align="start"
+        gap="1.25rem"
+        sm-gap="1rem"
+        justify="space-between"
+        class="quicklinks-content-container"
+      >
+        <v-text class="quicklinks-title" :weight="700" line-height="1.5">
+          {{ quickLink.title }}
+        </v-text>
+        <v-stack direction="row" align="center" justify="center" gap=".5rem">
+          <v-button
```

Comment:
in the component, only `button` value is checked to render button or anchor element though you pass `link`. 

so instead of using `type` property. can we implement logic where the component will check if there is `href` prop, if so it will render `<a>` or `<button`>

@ajithranka @shrinathprabhu

Reply:
Yeah we'll do that, the purpose of adding type was `button` and `a` have different styling. But yeah we can do that with href as well